collections/_services/coms.md

```diff
@@ -16,11 +16,10 @@ onboard:
 onboardDescription: We provide <em>Docker Images</em> for hosting your own Common Object Management Storage Service.
 urls:
   github: https://github.com/bcgov/common-object-management-service/
-  docs: https://coms-dev-master.apps.silver.devops.gov.bc.ca/api/v1/docs
-  showcase: https://comss-dev-master.apps.silver.devops.gov.bc.ca/app/
-  postman: /common-service-showcase/assets/files/COMS.postman_collection.json
+  docs: https://coms-dev.api.gov.bc.ca/api/v1/docs
```

Comment:
I'd recommend we pin this to the test endpoint instead of dev, but since we're likely revisiting this pretty soon after we actually go to production this can also work.